core/domain/exp_services_test.py

```diff
@@ -3150,130 +3148,6 @@ def test_loading_old_exploration_does_not_break_domain_object_ctor(self):
         self.assertEqual(exploration.to_yaml(), self.UPGRADED_EXP_YAML)
 
 
-class SuggestionActionUnitTests(test_utils.GenericTestBase):
```

Comment:
Where are the corresponding unit tests for the "True" case? These are important to ensure the whole system still works fine after the flag flip.

Reply:
These are tests for the old suggestion framework, which is deprecated. And we are basically at a point of no return to that framework (since we removed the flag that allowed us to go back to that framework). This is not intended to work with the flag set as true.

Reply:
OK -- please add inline comments specifying what exactly this test is testing that is no longer available in the new framework. (Be specific -- don't just say "new framework", "old framework" -- talk about exactly what calls are missing in the new framework that prevent this from being tested there.)

Reply:
Added a comment